addOns/spider/src/main/java/org/zaproxy/addon/spider/automation/SpiderJob.java

```diff
@@ -466,6 +466,7 @@ public static class Parameters extends AutomationData {
         private Boolean sendRefererHeader = true;
         private Integer threadCount = Constants.getDefaultThreadCount();
         private String userAgent = "";
+        private Boolean logoutAvoidance = false;
```

Comment:
Create a constant in the param and use in both places.

Reply:
?

Reply:
Create a constant for the default value, to use here and in https://github.com/zaproxy/zap-extensions/pull/6503/files#diff-275007d100d14bfe7bfd659f0c9ac0a525d837e76151a4730933cc195b266296R376